package.json

```diff
@@ -232,5 +232,8 @@
     "workbox-webpack-plugin": "5.1.4",
     "worker-loader": "^3.0.7"
   },
+  "resolutions": {
+    "@leafygreen-ui/side-nav/@leafygreen-ui/tooltip": "6.2.1"
+  },
```

Comment:
Whats this?

Reply:
In description! Basically prevents the accidental inclusion of Emotion 11 via `@leafygreen-ui/lib`, since this change was introduced in a patch update and the tooltip package doesn't have very specific dependencies pinned.